linux57-tkg/install.sh

```diff
@@ -164,6 +176,36 @@ if [ "$1" = "install" ]; then
       fi
     fi
   fi
+
+  # Replace dashes with underscores, it seems that it's being done by binrpm-pkg
+  # Se we can actually refer properly to the rpm files.
+  if [ "$_distro" = "Fedora" ]; then
+    _kernel_flavor=${_kernel_flavor//-/_}
+  fi
+
+  if [ "$_distro" = "Fedora" ]; then
+    if make -j ${_thread_num} binrpm-pkg EXTRAVERSION="_${_kernel_flavor}"; then
+    #if [ "0" = "0" ]; then
+      msg2 "Building successfully finished!"
+      read -p "Do you want to install the new Kernel ? y/[n]: " _install
+      if [ "$_install" = "y" ] || [ "$_install" = "Y" ] || [ "$_install" = "yes" ] || [ "$_install" = "Yes" ]; then
+        
+        _kernelname=$_basekernel.${_kernel_subver}_$_kernel_flavor
+        _headers_rpm="kernel-headers-${_kernelname}*.rpm"
+        _kernel_rpm="kernel-${_kernelname}*.rpm"
+        
+        sudo dnf install -y ~/rpmbuild/RPMS/x86_64/$_headers_rpm ~/rpmbuild/RPMS/x86_64/$_kernel_rpm
+        
+        msg2 "Install successful"
+
+        # Add to the list of installed kernels, used for uninstall
+        if ! { [ -f installed-kernels ] && grep -Fxq "$_kernelname" installed-kernels; }; then
+          cd "$_where"
+          echo $_kernelname >> installed-kernels 
+        fi   
+      fi
+    fi
+  fi
```

Comment:
Perhaps make this an `elif` with the Ubuntu installation, since they are analogous?

Reply:
Agreed. Will do.